demo_modules/gears/gears_server.js

```diff
@@ -111,7 +111,7 @@ export function load(debug, state, wallGeometry) {
           newZ = 1-newZ;
           // 2.5) Pick a random new radius (which might generate a random new
           // pitch), but that's okay.
-          newRadius = random.integer(35, 1000);
+          newRadius = random.integer(50, 1000);
```

Comment:
duplicated magic numbers? in my codebase?
make some consts, dude.

Reply:
Done.